source/hardware/controllers/denon_mc7000.rst

```diff
@@ -317,6 +321,53 @@ Use :hwlabel:`SHIFT` + pad button to stop a sampler while playing or eject a sam
 .. hint::
    To use default Mixxx behaviour and allow playing multiple samplers at the same time you can set the user variable ``MC7000.prevSamplerStop`` inside the :ref:`JavaScript file<denon_mc7000_uservariables>` to ``false``.
 
+Velocity Sampler Mode (purple LED)
+----------------------------------
+
+Velocity Sampler Mode is a mode where the velocity of the push on the pad determines the volume of the sampler played. 
+A gentle push results in low volume and a powerful push restults in high volume.
+Apart from that the behavior is equal to Sampler Mode.
+
+8 samplers can be triggered from each Deck if MC7000.SamplerQty is set to 32. Deck 1 will play sampler 1 to 8, Deck 2 will play sampler 9 to 16, Deck 3 will play sampler 17 to 24 and Deck 4 will play sampler 25 to 32.
+If MC7000.SamplerQty is set to 16, Deck 1 and Deck 3 will play the samplers 1 to 8 and Deck 2 and Deck 4 will play the samplers 9 to 16.
```

Comment:
I suggest to omit the description of the non-default behavior from the manual.

Reply:
16 samplers will be the standard and therefore all 32 sampler information be removed from the manual.